src/main/java/org/opentripplanner/routing/graph/GraphIndex.java

```diff
@@ -399,9 +399,10 @@ public BitSet servicesRunning (LocalDate date) {
      * @param startTime Start time for the search. Seconds from UNIX epoch
      * @param timeRange Searches forward for timeRange seconds from startTime
      * @param numberOfDepartures Number of departures to fetch per pattern
+     * @param omitNonPickups If departures with pickup restrictions should be included or not
```

Comment:
Since this is the key to understanding the parameter, I would still like to re-phrase it a little to avoid people misunderstanding it. How about "@param omitNonPickups If true, do not include vehicles that will not pick up passengers."